lesson15/script.js

```diff
@@ -85,6 +85,9 @@ console.log(sentence.replace('bad', 'good'));
 // 11. Repeat a String (H)
 // Create a visual separator for console output. Expected result: "--------------------"
 
+const expression = 'hey';
```

Comment:
Undo this change, it's a file the all the students use :)

Reply:
How can I undo a change? Is it the same with the above comment? "You need to change in the current branch, commit and push. DO NOT create a new pull request or branch."